tests/test_data_samples_order.py

```diff
@@ -77,9 +78,9 @@ def test_traintuple_data_samples_relative_order(factory, client, default_dataset
     traintuple = client.add_traintuple(spec)
     # Ensure the order of the data sample keys is correct at 2 levels: :
     #  1. In the returned traintuple
-    #  2. In the train method of the algo. If the order is incorrect, .future().wait() will fail.
+    #  2. In the train method of the algo. If the order is incorrect, Future(...).wait() will fail.
     assert traintuple.dataset.keys == data_sample_keys
-    traintuple.future().wait()
+    assets.Future(traintuple, client).wait()
```

Comment:
What do you think of wrapping the sdk response in Future not in the tests but rather in the Client directly?

Reply:
In retrospect, I'm not sure that wrapping a whole class is such a good idea. But maybe we could replace the class with a simpler function call ? `wait(traintuple, client)` or maybe `client.wait(traintuple)` ?

This doesn't change the fact that it's otherwise working fine!

Reply:
True, it can now be simplified to `client.wait`, no need for the whole `Future` class. I tried to wrap it to see how it was, did not see a clean way of doing it.  
Seeing how it was used in the code, I thought that this calling `sthg(my_asset).wait()` did not change the usage too much.